src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/FormattingExtensions.cs

```diff
@@ -53,7 +54,8 @@ public static bool ContainsElasticTrivia(this SuppressOperation operation, Token
         var endToken = tokenStream.GetTokenData(operation.EndToken);
         var previousToken = endToken.GetPreviousTokenData();
 
-        return tokenStream.GetTriviaData(startToken, nextToken).TreatAsElastic || tokenStream.GetTriviaData(previousToken, endToken).TreatAsElastic;
```

Comment:
getting TriviaData is not cheap.  It allocates up front (The substring between tokesn).  I looked into changing that, but it was non-trivial.  so thsi was the expedient solution.